tests/source/closure-block-labels.rs

```diff
@@ -0,0 +1,246 @@
+macro_rules! test_macro {
+    (|$transaction: ident| $body: expr) => {{
+        let $transaction = some_value;
+        let _ = $body;
+    }};
+}
+
+fn test_func<T>(func: impl FnOnce(u8) -> T) {
+}
+
+fn test_func2<T>(func: impl FnOnce(u8) -> T, value: u8) {
+}
```

Comment:
rustfmt can handle code that doesn't compile as long as it parses as valid rust. You can remove these helper functions and macro definitions.

Reply:
Fixed in 75b1cc42a64be93ba1646fac864700313a109b9d